packages/qwik/src/core/components/prefetch.ts

```diff
@@ -143,6 +143,13 @@ const PREFETCH_CODE = /*#__PURE__*/ ((
 export const PrefetchGraph = (
   opts: { base?: string; manifestHash?: string; manifestURL?: string; nonce?: string } = {}
 ) => {
+  const isTest = import.meta.env.TEST;
```

Comment:
same here, what is isTest?

Reply:
We have few tests for this component and we need to simulate the production mode even if we are testing